pycrunch/shoji.py

```diff
@@ -5,12 +5,17 @@
 """
 
 import json
+
+import time
 from six.moves import urllib
 
 import six
 
 from pycrunch import elements
-from pycrunch.lemonpy import URL
+from pycrunch.lemonpy import URL, ClientError, ServerError
+
+
+DEFAULT_PROGRESS_TIMEOUT = 30
```

Comment:
About `wait_for_batch_status` I left it only to cope with third party code that might be calling it directly. Should we remove it? 

We can also report a `DeprecationWarning` for now, but not many people run python with `-W` and it's probably not a good idea to mess up with other people warning filters to force our own warnings to always show up.